lib/aptible/auth/user.rb

```diff
@@ -14,7 +14,9 @@ class User < Resource
       field :updated_at, type: Time
 
       def organizations
-        roles.map(&:organization).uniq(&:id)
+        # Establish uniqueness of requests before loading all organizations
+        # We can do this by reading the `organization` link for each role
+        roles.map(&:links).map(&:organization).uniq(&:base_href).map(&:get)
```

Comment:
Does this not work properly if you use `href`?

The reason I ask is because the underlying library we use (HyperResource) has some support for templated URLs, and `base_href` would return the template, whereas the actual HREF that would be used for a `get` is present at `href`.

Since we don't currently use those templates, I'm fairly certain `href` should just passthrough to `base_href`:

```
  def href
    if self.templated?
      filtered_params = self.parent_resource.outgoing_uri_filter(params)
      URITemplate.new(self.base_href).expand(filtered_params)
    else
      self.base_href
    end
  end
```

But, assuming we can use this, using `href` is probably more robust.